CommunityToolkit.Mvvm.SourceGenerators/Messaging/IMessengerRegisterAllGenerator.cs

```diff
@@ -47,6 +48,8 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
             .Select(static (item, _) => Execute.GetInfo(item.Type, item.Interfaces))
             .WithComparer(RecipientInfo.Comparer.Default);
 
+        context.FilterWithLanguageVersion(ref recipientInfo, LanguageVersion.CSharp8, UnsupportedCSharpLanguageVersionError);
```

Comment:
Same here, this generator doesn't need the C# 8 gate, as it doesn't use any C# 8 features.
We can assume C# 7.0 as a baseline.

Reply:
If it doesn't need v8, then we should check against v7.3 (_for UWP_) as indicated in the comments of the attached issue.

Reply:
We can do that, but this is not the right place. I can do that as part of #432.